packages/nx/bin/nx.ts

```diff
@@ -93,3 +102,61 @@ function resolveNx(workspace: WorkspaceTypeAndRoot | null) {
     });
   }
 }
+
+/**
+ * Assumes currently running Nx is global install.
+ * Warns if out of date by 1 major version or more.
+ */
+function warnIfUsingOutdatedGlobalInstall(localNxVersion?: string) {
+  const globalVersion = require('../package.json').version;
+  const isOutdatedGlobalInstall =
+    globalVersion &&
+    ((localNxVersion && major(globalVersion) < major(localNxVersion)) ||
+      (!localNxVersion &&
+        getLatestVersionOfNx() &&
+        major(globalVersion) < major(getLatestVersionOfNx())));
+
+  // Using a global Nx Install
+  if (isOutdatedGlobalInstall) {
+    const bodyLines = localNxVersion
+      ? ['Its time to update Nx 🎉']
```

Comment:
This would show `Its time to update Nx 🎉` twice - once as a title and once as the first line in the body.